front/components/ElectedPoliticians/SinglePoliticianCard.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image';
+
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+import { User } from 'lucide-react';
+
+type TElectedPolitician = {
+  name: string;
+  photosrc: string;
+  fonction: string;
+  email: string;
+};
+
+export default function ElectedPolician({ name, photosrc, fonction, email }: TElectedPolitician) {
+  return (
+    <Card className='text-center'>
+      <CardHeader>
+        <CardTitle className='capitalize'>
+          {photosrc ? (
+            <img src={photosrc} width='140' height='140' alt='' className='mx-auto' />
```

Comment:
Les props comme les variables/constantes sont en camelCase !

```suggestion
          {photoSrc ? (
            <img src={photoSrc} width='140' height='140' alt='' className='mx-auto' />
```

Reply:
fixed